src/m365/spo/commands/list/list-retentionlabel-remove.ts

```diff
@@ -0,0 +1,175 @@
+import { Cli } from '../../../../cli/Cli';
+import { Logger } from '../../../../cli/Logger';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { formatting } from '../../../../utils/formatting';
+import { urlUtil } from '../../../../utils/urlUtil';
+import { validation } from '../../../../utils/validation';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+import { ListInstance } from './ListInstance';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+  listId?: string;
+  listTitle?: string;
+  listUrl?: string;
+  confirm: boolean;
+}
+
+class SpoListRetentionLabelRemoveCommand extends SpoCommand {
+  public get name(): string {
+    return commands.LIST_RETENTIONLABEL_REMOVE;
+  }
+
+  public alias(): string[] | undefined {
+    return [commands.LIST_LABEL_SET];
```

Comment:
This Alias can't be right here in this context, that is part of the ensure command

Reply:
oeps 😄